frontend/src/components/modals/ModalSuggestedTeachers.vue

```diff
@@ -7,13 +7,15 @@
     <h4>
       Teacher Name: <strong>{{ selectedTeacher.FullName }}</strong>
     </h4>
+    <h4 v-if="displayText"><br>Showing all teachers because there's no perfect match for this student<br></h4>
```

Comment:
This is might be possible and eliminate the need for the `displayText` variable, which is not very clear (by name) what text it is displaying. But this is a nitpick.

```suggestion
    <h4 v-if="!isSearchingTeachers && teachersData.length === 0"><br>Showing all teachers because there's no perfect match for this student<br></h4>
```